pytest_terraform/tf.py

```diff
@@ -50,17 +50,17 @@ def __init__(
         plugin_cache=None,
         stream_output=None,
         tf_bin=None,
+        env=None,
     ):
         self.work_dir = work_dir
         self.module_dir = module_dir
         # use parent dir of work/data dir to avoid
         # https://github.com/hashicorp/terraform/issues/22999
-        self.state_path = state_path or os.path.join(
-            work_dir, "..", "terraform.tfstate"
-        )
+        self.state_path = state_path or os.path.join(work_dir, "..", "terraform.tfstate")
         self.stream_output = stream_output
         self.plugin_cache = plugin_cache or ""
         self.tf_bin = tf_bin
+        self.env = env
```

Comment:
can we do assert (env is None or callable(env)), I'd rather fail early on a dictionary being passed. actually it would be preferable to have a dictionary here versus a callback if possible.

Reply:
Yeah, we can get by with just a dict and not a callable. Fixing now.